lms/views/api/exceptions.py

```diff
@@ -127,6 +127,7 @@ def external_request_error(self):
             },
         )
         sentry_sdk.set_context("validation_errors", self.context.validation_errors)
+        sentry_sdk.set_context("cause", self.context.__cause__)
```

Comment:
```suggestion
```

I tried this out in Sentry and it doesn't seem to work: you need to pass a dict to `set_context()` not an exception object, so it'd have to be something like  `sentry_sdk.set_context("cause", {"exc": repr(self.context.__cause__)})`. But I also noticed that Sentry seems to pull out the `__cause__` and the whole chain of exceptions automatically, so I actually don't think we need to do anything.

Reply:
For testing Sentry by the way: get your own free Sentry account, create a project, get the project's DSN, then do `export SENTRY_DSN='https://e7c5d3f2958a4c4781b11782931a8f86@o31866.ingest.sentry.io/1309830'` before running `make dev`

Reply:
Yes I think sentry get's the cause of the exception automatically:

https://sentry.io/organizations/hypothesis/issues/2959750441/?project=5952630&query=is%3Aunresolved